dashboard/app/helpers/course_block_helper.rb

```diff
@@ -115,7 +115,7 @@ def self.get_tall_course_block(id, family_name = nil)
       Unit::DANCE_PARTY_2019_NAME => {
         body: data_t_suffix('script.name', id, 'description_short'),
         url: CDO.code_org_url('/dance'),
-        image_url: CDO.shared_image_url("courses/logo_tall_#{id}.gif")
+        image_url: CDO.shared_image_url("courses/logo_tall_dance-2022.png")
```

Comment:
We decided to simply swap this image instead of creating a `Unit::DANCE_PARTY_2022_NAME` constant.